### ISSUE TITLE
Documentation Request

### ISSUE THREAD

**Issue by sumonsm** [camsys/transam_core#117]
Hi, I am a Rails developer. I am trying to install and setup the whole transam system in my VM:
```
TransAM CORE Application: https://github.com/camsys/transam_core
Transit focused engine for TransAM: https://github.com/camsys/transam_transit
Capital Planning Tool (CPT) Engine for TransAM: https://github.com/camsys/transam_cpt
Accounting Extensions for TransAM: https://github.com/camsys/transam_accounting
Traffic Sign Extensions for TransAM: https://github.com/camsys/transam_sign
Maintenance Management System Extensions for TransAM: https://github.com/camsys/transam_mms
Audting extensions for TransAM: https://github.com/camsys/transam_audit
Document Library for TransAM: https://github.com/camsys/transam_lib
```

But I am currently having a hard time running the migrations successfully. Is there any way you can help me with this? I installed the dependency. But I cant run the migrations successfully. I tried adding the engines to a Rails 4.1.8 application one by one, starting with transam_core. But I keep getting errors. The first migration tries to modify a table. What is the correct order of getting all of these engines setup? Is there a standard setup documentation available? Please help. Thanks for reading.

**Comment by smeeks** [comment]
Closing this very old issue as part of general cleanup. TransAM is not yet to the point of providing an independent developer's guide.

**Comment by vikasnautiyal** [comment]
Hello,

I'm trying to install and use the TransAMCore gem in my Rails project, but I encountered some challenges regarding the documentation. Specifically, the documentation doesn't explicitly mention the required Ruby version for TransAMCore.

I ran `bundle install` and encountered the following error message:

```
Bundler could not find compatible versions for gem "ruby":
  In Gemfile:
    ruby (~> 2.5.3.0)

    transam_core was resolved to 2.46.0.pre.rc, which depends on
      nokogiri (>= 1.13.4) was resolved to 1.13.10, which depends on
        ruby (>= 2.6.0)
```

Based on this error, it seems TransAMCore might have a minimum Ruby version requirement of 2.6.0. It would be helpful if the documentation could be updated to explicitly state the supported Ruby version range for TransAMCore installation. Or if you may want I can create a PR updating the .[transam_core.gemspec (https://github.com/camsys/transam_core/blob/develop/transam_core.gemspec)

Additionally, could you please clarify the process for mounting TransAMCore in a Rails application? I'm hoping for some additional guidance on mounting TransAMCore as an engine. Could you please elaborate on the process?

Thanks for your time and assistance. I appreciate your work on TransAMCore!
